core/src/main/java/com/g16/feyrune/model/Model.java

```diff
@@ -20,12 +20,12 @@ public class Model {
     public Model() {
         player = new Player("Player", new Point(20, 5)); //TODO: Should probably have a method to get the initial player position from the map
         this.overworldModel = new OverworldModel(player);
+        this.overworldModel.addObserver(this);
         this.observers = new ArrayList<>();
         stateHandler = new StateHandler(ModelState.WORLD);
     }
 
     public void update() {
```

Comment:
why is this function empty?

Reply:
It should update the model of the current state